include/psa/crypto_values.h

```diff
@@ -316,6 +316,9 @@
     (((type) & PSA_KEY_TYPE_CATEGORY_MASK) == PSA_KEY_TYPE_CATEGORY_RAW || \
      ((type) & PSA_KEY_TYPE_CATEGORY_MASK) == PSA_KEY_TYPE_CATEGORY_SYMMETRIC)
 
+/** Whether a key type is symmetric. */
+#define PSA_KEY_TYPE_IS_SYMMETRIC(type)                                 \
```

Comment:
Same as above.

Reply:
Done

Reply:
This macro is no longer added by this PR (it was removed in a later commit in response to this comment), so I'm marking this conversation as resolved.